### ISSUE TITLE
json-patch schema can't be even loaded

### ISSUE THREAD

**Issue by s13o** [everit-org/json-schema#174]
Just try to load a json schema from http://json.schemastore.org/json-patch by 'SchemaLoader.load'
The exception is 
```
org.everit.json.schema.SchemaException: #/definitions/operation: expected at most 1 of 'allOf', 'anyOf', 'oneOf', 2 found
```

**Comment by erosb** [comment]
Sad. If it helps, rewriting the schema this way should work (didn't test it):

```
{
    "$schema": "http://json-schema.org/draft-04/schema#",
    "definitions": {
        "operation": {
            "allOf": [
                {
                    "allOf": [
                        {
                            "$ref": "#/definitions/path"
                        }
                    ]
                },
                {
                    "oneOf": [
                        {
                            "properties": {
                                "op": {
                                    "description": "The operation to perform.",
                                    "enum": [
                                        "add",
                                        "replace",
                                        "test"
                                    ],
                                    "type": "string"
                                },
                                "value": {
                                    "description": "The value to add, replace or test."
                                }
                            },
                            "required": [
                                "value"
                            ]
                        },
                        {
                            "properties": {
                                "op": {
                                    "description": "The operation to perform.",
                                    "enum": [
                                        "remove"
                                    ],
                                    "type": "string"
                                }
                            }
                        },
                        {
                            "properties": {
                                "from": {
                                    "description": "A JSON Pointer path pointing to the location to move/copy from.",
                                    "type": "string"
                                },
                                "op": {
                                    "description": "The operation to perform.",
                                    "enum": [
                                        "move",
                                        "copy"
                                    ],
                                    "type": "string"
                                }
                            },
                            "required": [
                                "from"
                            ]
                        }
                    ]
                }
            ],
            "required": [
                "op",
                "path"
            ],
            "type": "object"
        },
        "path": {
            "properties": {
                "path": {
                    "description": "A JSON Pointer path.",
                    "type": "string"
                }
            }
        }
    },
    "items": {
        "$ref": "#/definitions/operation"
    },
    "title": "JSON schema for JSONPatch files",
    "type": "array"
}
```

Fixing the real issue (properly loading multiple combined schemas at the same nesting level) might not come any soon.

**Comment by erosb** [comment]
The fix is available in the latest release: [1.9.0](https://github.com/everit-org/json-schema/releases/tag/1.9.0)
